Scripts/DCS-BIOS/lib/modules/aircraft_modules/F-4E.lua

```diff
@@ -1452,6 +1452,38 @@ local AIRCRAFT_EFFECTS_DEVICE_ID = 63
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_BORESIGHT_AZIMUTH", TARGET_DESIGNATOR_DEVICE_ID, 3002, 2734, { 0, 1 }, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Boresight Azimuth (12-Vis)")
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_BORESIGHT_ELEVATION", TARGET_DESIGNATOR_DEVICE_ID, 3003, 2735, { 0, 1 }, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Boresight Elevation (12-Vis)")
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_BORESIGHT_ROLL", TARGET_DESIGNATOR_DEVICE_ID, 3004, 2736, { 0, 1 }, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Boresight Roll (9-Vis)")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_STOW", TARGET_DESIGNATOR_DEVICE_ID, 3005, 2754, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Stow/Unstow TGP")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_LASER_READY", TARGET_DESIGNATOR_DEVICE_ID, 3006, 2755, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Arm/Disarm Laser")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_POWER_ON", TARGET_DESIGNATOR_DEVICE_ID, 3007, 2756, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Power On/Off TGP")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_BIT", TARGET_DESIGNATOR_DEVICE_ID, 3008, 2737, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Next BIT")
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_LIGHT_BRIGHTNESS", TARGET_DESIGNATOR_DEVICE_ID, 3009, 2738, { 0, 1 }, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Panel Light Brightness")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_REJECT_OVERRIDE", TARGET_DESIGNATOR_DEVICE_ID, 3010, 2739, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Reject/Override Range")
+F_4E:define3PosTumb0To1("WSO_TGT_DESIGNATOR_ACQUISITION", TARGET_DESIGNATOR_DEVICE_ID, 3011, 2740, 3, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Acquisition Mode")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_WRCS_OUT", TARGET_DESIGNATOR_DEVICE_ID, 3012, 2757, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Enable/Disable WRCS integration")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_GO_MALF", TARGET_DESIGNATOR_DEVICE_ID, 3013, 2758, WSO_TARGET_DESIGNATOR_SET_CONTROL, "(no function)")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_INS_OUT", TARGET_DESIGNATOR_DEVICE_ID, 3014, 2759, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Enable/Disable INS integration")
+
+-- Laser Coder Control
+F_4E:definePushButton("WSO_LASER_CODE_ENTER", TARGET_DESIGNATOR_DEVICE_ID, 3015, 2504, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Enter Laser Code")
+F_4E:definePushButton("WSO_LASER_CODE_SET_THOUSANDS", TARGET_DESIGNATOR_DEVICE_ID, 3019, 2500, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Next Laser Code (thousands)")
+F_4E:definePushButton("WSO_LASER_CODE_SET_HUNDREDS", TARGET_DESIGNATOR_DEVICE_ID, 3018, 2501, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Next Laser Code (hundreds)")
+F_4E:definePushButton("WSO_LASER_CODE_SET_TENS", TARGET_DESIGNATOR_DEVICE_ID, 3017, 2502, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Next Laser Code (tens)")
+F_4E:definePushButton("WSO_LASER_CODE_SET_ONES", TARGET_DESIGNATOR_DEVICE_ID, 3016, 2503, WSO_TARGET_DESIGNATOR_SET_CONTROL, "Next Laser Code (ones)")
```

Comment:
If you wanted to break this into a separate category like "WSO Laser Code Panel" or something that would make sense to me - the categories don't have to strictly match the device ID. I'm not familiar enough with the WSO seat in the F-4 to know if that makes sense though.

Reply:
I'm not familiar with Phantom yet as well, but this make sense [Laser system](https://f4.manuals.heatblur.se/systems/weapon_systems/pave_spike/laser_system.html#laser-system)